apps/src/aiTutor/types.ts

```diff
@@ -32,6 +32,18 @@ export type StudentChatRow = {
   createdAt: string;
 };
 
+export type StudentServerData = {
+  id: number;
+  name: string;
+  ai_tutor_access_denied: boolean;
+};
+
+export type StudentAccessData = {
+  id: number;
+  name: string;
+  aiTutorAccessDenied: boolean;
```

Comment:
Are these two (very similar and seemingly redundant) types the best way to handle the needs of this function? https://github.com/code-dot-org/code-dot-org/pull/57073/files#diff-5acb1a07ce361937102b013abfe304d2f83a9e68f95a1fc9272ab39fe5761b60R6

Reply:
They are pretty similar... tho its not _totally_ crazy to use a separate model for the react hooks either 🤷‍♂️ My favorite code is code that works 🤣.

Reply:
I'll defer to @sanchitmalhotra126.